### PR TITLE
834328-Added "How to" for MaskedTextBox control to restrict validation

### PR DESCRIPTION
**Pull request by BhaskarSuresh22**
Added "How to" section for restricting validation support in WinUI MaskedTextBox control.

### CODE REVIEW

File: winui/Masked-TextBox/How-To.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+layout: post
+title: Validation support in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the validation support in the MaskedTextBox (SfMaskedTextBox) control.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# How to restrict the validation in WinUI MaskedTextBox
+
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/How-To.md:11]: @BhaskarSuresh22 How to restrict the default validation in WinUI MaskedTextBox

File: winui/Masked-TextBox/How-To.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Validation support in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the validation support in the MaskedTextBox (SfMaskedTextBox) control.
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/How-To.md:4]: @BhaskarSuresh22 Learn all about the restrict the validation support in the MaskedTextBox (SfMaskedTextBox) control.

File: winui/Masked-TextBox/How-To.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# How to restrict the validation in WinUI MaskedTextBox
+
+The MaskedTextBox control provides the ability to validate the input according to the completion of masked input. By default, validation is enabled in the MaskedTextBox control. However, you can disable the validation by assigning `true` to the IsValid property of the ValueChanging event.
+
+{% tabs %}
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/How-To.md:14]: @BhaskarSuresh22 `true` to the `IsValid`

File: winui/Masked-TextBox/How-To.md (hunk 1 above)
Comment (BhaskarSuresh22) [reply winui/Masked-TextBox/How-To.md:11]: @Manivannan-E, Done the changes.

File: winui/Masked-TextBox/How-To.md (hunk 2 above)
Comment (BhaskarSuresh22) [reply winui/Masked-TextBox/How-To.md]: @Manivannan-E, Done the changes.

File: winui/Masked-TextBox/How-To.md (hunk 3 above)
Comment (BhaskarSuresh22) [reply winui/Masked-TextBox/How-To.md:14]: @Manivannan-E, Done the changes.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**